Description: doc: sphinx: explicitly set the doc language
 Needed for Sphinx 5.0 compatibility.
Author: Paride Legovini <paride@debian.org>
Origin: vendor
Bug: https://gitlab.isc.org/isc-projects/kea/-/issues/2451
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1013407
Last-Update: 2022-06-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/doc/sphinx/conf.py
+++ b/doc/sphinx/conf.py
@@ -77,7 +77,7 @@
 #
 # This is also used if you do content translation via gettext catalogs.
 # Usually you set "language" from the command line for these cases.
-language = None
+language = 'en'
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
